### ISSUE TITLE
Need support for configurable imports

### ISSUE THREAD

**Issue by mkustermann** [dart-lang/kernel#29]
Flutter depends on mojo, which already uses configurable imports. See [this code](https://github.com/flutter/buildroot/blob/master/mojo/dart/packages/mojo/lib/src/internal.dart#L10):

```dart
library mojo_internal;

// Import 'internal_contract.dart' by default, but use 'dart:mojo.internal' if
// the embedder supports it.
export 'internal_contract.dart'
  if (dart.library.mojo.internal) 'dart:mojo.internal';
```

**Comment by asgerf** [comment]
Support for this landed in the analyzer recently, but is not yet on the pub version.

**Comment by bwilkerson** [comment]
Do you need a new published version of the 'analyzer' package, or can you just point to a recent commit on github?

**Comment by asgerf** [comment]
Should be resolved as of b99158d739a189898ea858d210d1af327a971a9b.

Thanks again, Brian.
